jwst/ramp_fitting/tests/test_ramp_fit.py

```diff
@@ -0,0 +1,168 @@
+-import pytest
```

Comment:
Looks like the leading "-" at the beginning of this line is a typo and causing Travis to crash.